Tests/FoundationTests/URLRequestExtensionsTests.swift

```diff
@@ -14,6 +14,9 @@ import Foundation
 
 final class URLRequestExtensionsTests: XCTestCase {
 
+    private let insightNASAcURL = "curl https://api.nasa.gov/insight_weather/?api_key=mxbd8VDIy5CheCbrYgknXVH6X9ElpQaHMhne2YXP&feedtype=json&ver=1.0"
```

Comment:
Can we move decl to local `testCUrlSring` in the function?

Reply:
Yep, don't even know why I decide to do like that